### PR TITLE
Timezone

### LINKED ISSUE #7630: Time series graphs no longer show in the user's selected timezone
Graphs are showing the timestamps in the tooltip on the system timezone, rather than the user's configured timezone (in my case, graphs are displaying in PDT rather than UTC.)

### PR DESCRIPTION
**Pull request by lynnagara**
Fixes https://github.com/getsentry/sentry/issues/7630

### CODE REVIEW

File: src/sentry/static/sentry/app/components/stackedBarChart.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import createReactClass from 'create-react-class';
 import PropTypes from 'prop-types';
-import moment from 'moment';
+import moment from 'moment-timezone';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/stackedBarChart.jsx]: Do you need to import from 'moment-timezone'? You're not doing anything with `moment.tz`

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/stackedBarChart.jsx]: The timezone is probably configured into moment timezone somewhere, and this is just making the usage timezone aware.

Comment (lynnagara) [reply src/sentry/static/sentry/app/components/stackedBarChart.jsx]: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/stores/configStore.jsx#L32

Comment (lynnagara) [reply src/sentry/static/sentry/app/components/stackedBarChart.jsx]: `moment()` just displays in UTC

Comment (billyvg) [reply src/sentry/static/sentry/app/components/stackedBarChart.jsx]: Ah, (unrelated to this PR), should we update most of our other `moment` imports too?

**Review by billyvg — APPROVED** [review]

**Pull request merged**